crates/ruff_python_ast/src/scope.rs

```diff
@@ -43,9 +49,33 @@ impl<'a> Scope<'a> {
         self.bindings.get(name)
     }
 
+    /// Returns the [id](BindingId) of the binding with the given name.
+    pub fn lookup(&self, qualified_name: &str) -> Option<&BindingId> {
+        self.imports.get(qualified_name)
+    }
```

Comment:
This is the API I'm trying to enable: look up a fully qualified name without having to iterate over all bindings.